projects/can_api/utils.py

```diff
@@ -1,14 +1,18 @@
 def get_rx_messages(subs, messages):
     """
-    Gets the received Messages from the list of messages, based on the subs 
+    Gets the received Messages from the list of messages, based on the subs
     dictionary.
     """
     rx_message_names = [d["name"] for d in subs]
     rx_messages = list(filter(lambda m: m.name in rx_message_names, messages))
+    masks = {
```

Comment:
Suggestion: consider using `subscriber.get('mask', 0x7ff)` to replace the inline if for brevity and readability

Also, can we rename from `subscriber` to `message` or `msg`, since that's a more accurate depiction of what that object is?

Reply:
Ah completely agree with the first suggestion. As for the second one, isn't subscriber the appropriate name since I'm looping through subs, which is a list of potential subscribers that CAN messages can be received from?

Reply:
the MCU itself is the subscriber. each entry would be a subscription. i like Amit’s idea of using message.